covid19-etl/etl/cityofchicago.py

```diff
@@ -354,36 +367,44 @@ def files_to_submissions(self):
         start = time.time()
         latest_submitted_date = self.metadata_helper.get_latest_submitted_date_idph()
 
-        # The following condition is for the first entry in dataset, which is from date `03-01-2020`
-        if latest_submitted_date == None:
-            latest_submitted_date = datetime(2020, 3, 1)
-
         today = datetime.today().replace(hour=0, minute=0, second=0, microsecond=0)
 
-        if latest_submitted_date == today:
+        if latest_submitted_date != None and latest_submitted_date == today:
```

Comment:
`if latest_submitted_date == today:` did the same thing as `if latest_submitted_date != None and latest_submitted_date == today:` 🤔

Reply:
I thought I would require an explicit condition to check. Editing this